src/Algebra/Consequences/Setoid.agda

```diff
@@ -29,6 +29,40 @@ open import Relation.Unary using (Pred)
 
 open import Algebra.Consequences.Base public
 
+------------------------------------------------------------------------
+-- Congruence and MiddleFourExchange
```

Comment:
Likewise, I'm not sure `Congruence` should be in the title here. The proofs aren't about congruence per se.